content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -16,10 +15,37 @@ This quickstart requires:
 * An AWS account with permissions to create an S3 storage bucket
 * AWS [access keys](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds)
 
+## About Managed Resources in Crossplane v2
+A _managed resource_ is anything Crossplane creates and manages outside of the
+Kubernetes cluster.
+
+This guide creates an AWS S3 bucket with Crossplane.
+
+The S3 bucket is a _managed resource_.
+
+Crossplane v2 allows you to compose namespaced resources. To better support this
+new ability, managed resources (MRs) are now namespaced in Providers that have
+been updated for Crossplane v2.
+
+To support backwards compatibility while users are adopting Crossplane v2, each
+provider will offer the legacy cluster scoped MRs in addition to the new
+namespaced MRs.
+
+For example, when the AWS provider that has been upated to support Crossplane v2
+is installed during this guide, you will see two CRDs for each type of managed
+resource:
+
+1. A legacy cluster scoped MR in the `*.aws.upbound.io` API group
+1. A namespaced MR in the `*.aws.m.upbound.io` API group
+
+{{< hint type="tip" >}}
+More about namespaced managed resources can be read in the [Crossplane v2 proposal](https://github.com/crossplane/crossplane/pull/6255).
+{{< /hint >}}
```

Comment:
I'd actually prefer to leave this detail out (in this guide at least).

Instead I'd like to completely ignore legacy topics, and just show a v2 style namespaced MR in this guide. That way folks who aren't coming from the legacy Crossplane v1 world won't need to worry about any of this.

Or another way to put it: write the v2 preview documentation as if Crossplane v1 never existed.

For folks who _are_ coming from the legacy Crossplane v1 world I'd like to separately add a "What's new in Crossplane v2" page that explains things like this. I'm tentatively thinking we could add that under Guides, but (if possible) update the "This isn't the latest version" header to link to that guide if they're looking at a v2 page.

Reply:
Got it, thanks for seeing that tone to follow here, will incorporate that.